### PR TITLE
Update drupal.org URL

### PR DESCRIPTION
**Pull request by mukeshpanchal27**

### CODE REVIEW

**Comment by mukeshpanchal27** [comment]
@schlessera can you please check this

File: bin/wp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #
 # This wrapper script has been adapted from the equivalent drush wrapper
 # and 99.9% of all credit should go to the authors of that project:
-# http://drupal.org/project/drush
+# https://drupal.org/project/drush
```
Comment (swissspidy) [inline bin/wp]:
```suggestion
# https://www.drupal.org/project/drush
```

File: bin/wp [hunk 2]
```diff
@@ -27,7 +27,7 @@ if [ ! -z "$WP_CLI_PHP" ] ; then
 	php="$WP_CLI_PHP"
 else
 	# Default to using the php that we find on the PATH.
-	# Note that we need the full path to php here for Dreamhost, which behaves oddly.  See http://drupal.org/node/662926
+	# Note that we need the full path to php here for Dreamhost, which behaves oddly.  See https://drupal.org/node/662926
```
Comment (swissspidy) [inline bin/wp]:
```suggestion
	# Note that we need the full path to php here for Dreamhost, which behaves oddly.  See https://www.drupal.org/node/662926
```

**Comment by schlessera** [comment]
Thanks for the PR, @mukeshpanchal27 !

**Pull request merged**